packages/ses/src/scope-handler.js

```diff
@@ -162,7 +159,7 @@ export const createScopeHandler = (
 
     getOwnPropertyDescriptor(_target, prop) {
       // Coerce with `String` in case prop is a symbol.
-      const quotedProp = JSON.stringify(String(prop));
+      const quotedProp = q(String(prop));
```

Comment:
This is another incidental refactor for quoting. I don’t know whether we need the inner `String` for coercing symbols anymore.

Reply:
See https://github.com/endojs/endo/blob/f07a8b3fa670a219e06f1a7ba04524d50de85a1e/packages/ses/src/error/stringify-utils.js#L101
It is not quite equivalent. But if you agree that it is adequately equivalent, then
```suggestion
      const quotedProp = q(prop);
```